section4/exercises/mycar.rb

```diff
@@ -0,0 +1,45 @@
+class MyCar
+  attr_accessor :name , :color , :model
+  attr_reader :year
+
+  def initialize(n, y, c, m)
```

Comment:
I'd advise against getting in the habit of using 1 letter variables/params. In this case, it feels reasonable/easy enough to follow but for 99% of your time writing Ruby and most modern languages, it's a no-no - reason being is readability for other devs!